ui-tests/src/test/resources/features/wip/data-buckets.feature

```diff
@@ -161,3 +162,215 @@ Feature: Test functionality of data buckets during creation of integration
+
+    # first database connection - get some information to be used by second datamapper
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Periodic SQL Invocation" integration action
+    Then she checks "Done" button is "Disabled"
+    Then she fills periodic query input with "select company as firma from contact limit 1;" value
+    Then she fills period input with "10000" value
+
+    And clicks on the "Done" button
+
+
+    ##################### finish step ##################################
+    Then she is prompted to select a "Finish" connection from a list of available connections
+
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL" integration action
+    Then she fills invoke query input with "select * from contact where company = :#COMPANY and first_name = :#MYNAME and last_name = :#MYSURNAME and lead_source = :#LEAD" value
+    And clicks on the "Done" button
+
+
+
+    ##################### step NEW step ##################################
+    And She adds integration "connection" on position "0"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL" integration action
+    Then she fills invoke query input with "select first_name as myName from contact where company = :#COMPANY and last_name = :#MYSURNAME limit 1;" value
+    And clicks on the "Done" button
+
+
+    ##################### step NEW step step ##################################
+
+    And She adds integration "connection" on position "0"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL" integration action
+    Then she fills invoke query input with "select last_name as mySurname from contact where company = :#COMPANY limit 1;" value
+    And clicks on the "Done" button
+
+    ##################### step step step NEW step ##################################
+
+    And She adds integration "connection" on position "2"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL" integration action
+    Then she fills invoke query input with "select lead_source as myLeadSource from contact where company = :#COMPANY and first_name = :#MYNAME and last_name = :#MYSURNAME limit 1;" value
+    And clicks on the "Done" button
+
+    ##################### step step step step NEW step ##################################
+
+    And She adds integration "connection" on position "3"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL" integration action
+    Then she fills invoke query input with "insert into contact values (:#COMPANY , :#MYNAME , :#MYSURNAME , :#LEAD, '1999-01-01')" value
+    And clicks on the "Done" button
+
+    ##################### check warnings ##################################
+
+    And she checks that there is no warning inside of step number "0"
+
+    Then she checks that text "Add a data mapping step" is "visible" in step warning inside of step number "2"
```

Comment:
Also a range here would be much nicer && clearer. Feel free to get rid off `she` subject. `The check text...` is sufficient imho.

Reply:
This step will not be used outside of this feature. And even if so, it would be without range as steps you may want to check can be with odd/even or random indexes, depends if you already have data mapper or not. I think it is clear enough but if you insist I will create range for this specific case so it is just one row in feature file instead of 5.

Reply:
@mcada sure it doesn't have to be range, but maybe an array of steps to check. You got me here I want to get rid off those lines...